packages/dds/tree/package.json

```diff
@@ -11,18 +11,42 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/tree-public.d.ts",
```

Comment:
Suggest making this change separately.

Reply:
Are you sure? We want this make sure all is good downstream with the exports here. Having this helped me validate (and then fix things).